src/styles/index.scss

```diff
@@ -1,5 +1,69 @@
-@import 'reset';
-@import 'helpers/style';
-@import 'fonts/style';
-@import 'normalize';
-@import 'components/style';
+@import 'utilities/mixins/style';
+
+// Variables and override
+@import 'utilities/helpers/bootstrap/variables';
+@import 'gommette/helpers/utils';
+@import 'gommette/helpers/variables';
+@import '../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/variables';
```

Comment:
Try to use node_modules/bootstrap-sass/assets/stylesheets/bootstrap/variables directly, without path points